pom.xml

```
@@ -21,10 +21,12 @@
         <module>configmap/file-system</module>
         <module>config-secret/api-server</module>
         <module>config-secret/file-system</module>
+        <!-- The messaging tests and integration tests are currently flaky and require investigation.
+             Disabling them until further notice.
         <module>messaging/artemis</module>
         <module>messaging/artemis-jta</module>
         <module>messaging/amqp-reactive</module>
-        <module>messaging/qpid</module>
+        <module>messaging/qpid</module>-->
```

```suggestion
        <module>messaging/qpid</module>
        -->
```

It makes the diff nicer, both now and when it's reverted :-)

---

Done!